src/decorators/sub-store.ts

```diff
@@ -0,0 +1,17 @@
+import { Reducer } from 'redux';
+
+export interface IFractalStoreOptions {
+  basePathMethodName: string
+  localReducer: Reducer<any>;
+}
+
+export const SubStore = ({
+  basePathMethodName,
+  localReducer
+}: IFractalStoreOptions) =>
+  (constructor: Function): void => {
+    constructor['@angular-redux::fractal-store::options'] = {
```

Comment:
Instead of putting a property on it this way, should probably use the reflect-metadata api. 

Something like (pseduo-code) ....

```ts
const METADATA_KEY =  '@angular-redux::fractal-store::options'


export const SubStore = ( .... ) => (constructor: Function): void => {
 if (!(Reflect as any).hasOwnMetadata(METADATA_KEY, target)) {
      (Reflect as any).defineMetadata(METADATA_KEY, [], target);
    }
   const theDatas:[] = (Reflect as any).getOwnMetadata(METADATA_KEY, target);
   const metadata: EpicMetadata = {/* things*/ };
    (Reflect as any).defineMetadata(METADATA_KEY, [...theDatas, metadata], target);
}
```

(some rough copy-pasta from the `@Epic` decorator)